### PR TITLE
adjustment of names and breaking functions out

### PR DESCRIPTION
**Pull request by crutkas**
## PR checklist

Quick summary of changes

- Which issue does this PR relate to?
#966 
- Anything that requires particular review or attention?
nope
- Do all automated tests pass?
yup

### CODE REVIEW

File: code/test/Core.Test/Naming/NamingTest.cs [hunk 1]
```diff
@@ -207,24 +207,37 @@ public void Validate_BadFormat_StartWithNumber(string language)
...
+        public void Validate_DirectoryExistsValidator_NonExistantFolderDirectory()
         {
-            var directoryValidator = new DirectoryExistsValidator(Environment.CurrentDirectory);
-            var invalidDirectoryValidator = new DirectoryExistsValidator(Path.Combine(Environment.CurrentDirectory, Guid.NewGuid().ToString()));
-            var existingDir = Directory.CreateDirectory(Guid.NewGuid().ToString());
+            var nonExistantFolderDirectoryValidator = new DirectoryExistsValidator(Path.Combine(Environment.CurrentDirectory, Guid.NewGuid().ToString()));
 
             // invalid dir
-            var invalidDirectory = invalidDirectoryValidator.Validate(Guid.NewGuid().ToString());
+            var invalidDirectory = nonExistantFolderDirectoryValidator.Validate(Guid.NewGuid().ToString());
```
Comment (mrlacey) [inline code/test/Core.Test/Naming/NamingTest.cs]: That an "invalid directory" validates as valid suggests something is wrong in this test or the name of the underlying class is very misleading.

**Comment by crutkas** [comment]
adjusted naming.  FYI, i didn't make the class, i just am adding in unit tests so it doesn't blow up like it did prior.

**Comment by crutkas** [comment]
@mrlacey  i adjusted

**Comment by mrlacey** [comment]
> FYI, i didn't make the class, i just am adding in unit tests so it doesn't blow up like it did prior.

Doesn't sound like a good reason not to fix it. We're here, looking at the code so we should address any issues we find. Or, at the very least raise an issue to have someone come back and address them later.

**Comment by crutkas** [comment]
@mrlacey i would argue that is outside the scope of this code review.  If you dislike the name of the DirectoryExistsValidator class, 100% lets adjust it, just that is a different issue.  I'm just trying to add test coverage for something that didn't have it before and caused a hard crash before the fix.

**Pull request merged**